content/cpp/concepts/maps/terms/empty/empty.md

```diff
@@ -0,0 +1,97 @@
+---
+Title: 'empty'
+Description: 'Checks to see if an STL map object is empty (contains elements) or not and returns a boolean true or false value, respectively'
+Subjects: 
+  - 'Computer Science'
+Tags:
+  - 'Cpp'
+  - 'Data Structures'
+  - 'Documentation'
+  - 'Functions'
+  - 'Map'
+  - 'Programming'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The C++ Standard Template Library (STL) [map](https://www.codecademy.com/resources/docs/cpp/maps) class **.empty()** function is a member function of the STL map class. This function checks to see if an STL map object is empty (contains no key-value pairs) or not and returns a boolean true or false value, respectively
```

Comment:
```suggestion
The C++ Standard Template Library (STL) [map](https://www.codecademy.com/resources/docs/cpp/maps) class **`.empty()`** function is a member function that checks if an STL map object contains no key-value pairs. It returns a boolean `true` if the map is empty and `false` otherwise.
```